src/test/java/guitests/UseGlobalConfigsTest.java

```diff
@@ -56,6 +70,31 @@ public void globalConfigTest() {
         press(KeyCode.SHIFT).press(KeyCode.SEMICOLON).release(KeyCode.SEMICOLON).release(KeyCode.SHIFT);
         type("dummy2/dummy2");
         push(KeyCode.ENTER);
+
+        click("#dummy/dummy_col1_renameButton");
+        type("Dummy 2 panel");
+        PanelNameTextField renameTextField2 = find("#dummy/dummy_col1_renameTextField");
+        assertEquals("Dummy 2 panel", renameTextField2.getText());
+        push(KeyCode.ENTER);
+        
+        // Creating panel to the left
+        press(KeyCode.SHIFT).press(KeyCode.CONTROL).press(KeyCode.P);
```

Comment:
It may be good to abstract actions like this to utility methods. e.g. `createPanelOnLeft()` which makes the code easier to read and maintain.